.github/ISSUE_TEMPLATE/question.md

```diff
@@ -1,6 +1,6 @@
 ---
 name: ❓ Questions/Help
-about: If you have questions, please check Aragon Hack, Aragon Forum or Aragon Chat
+about: If you have questions, please check the Aragon Spectrum Community, Aragon Hack or Aragon Forum.
```

Comment:
I wonder if we can add links for this?

Reply:
Not in the "about" section but links are available a little bit further down the issue template: https://github.com/aragon/aragon-cli/pull/1553/files#diff-1920b7226ceeed1523ae3615db9817f3R12